src/main/scala/cromwell/binding/values/WdlString.scala

```diff
@@ -4,7 +4,13 @@ import cromwell.binding.types.WdlStringType
 import org.apache.commons.lang3.StringEscapeUtils
 import scala.util.{Success, Try}
 
-case class WdlString(value: String) extends WdlPrimitive {
+trait WdlStringLike {
+  def value: String
+  def isGcsUrl: Boolean = value.startsWith("gs://")
```

Comment:
I thought if you did read_string("file.txt") the "file.txt" got coerced into a WdlFile before being passed to read_string?
I didn't think literal GCS paths were supported in WDL like that - won't they get "re-GCSified"?

But my assumptions on both of the above might be incorrect!